drv/fe310-rtc/src/main.rs

```diff
@@ -5,19 +5,28 @@
 #![no_std]
 #![no_main]
 
+use drv_riscv_plic_api::*;
 use ringbuf::*;
 #[allow(unused_imports)]
 use userlib::*;
 
+task_slot!(INT_CONTROLLER, ext_int_ctrl);
+
 #[export_name = "main"]
 fn main() -> ! {
     const RTC_INT: u32 = 0x1;
     ringbuf!(u64, 64, 0);
 
-    sys_irq_control(RTC_INT, true);
```

Comment:
This change works fine for the current driver as it is RV specific. If we ever run into a case where a driver/task is shared by RV and ARM, then it could get really messy to handle the calls differently for each. Anyways, we don't have to worry about that now, but just something to keep in mind for the future.

Reply:
I agree. I think it _is_ possible to make that work, but it would require some changes to the syscall internals that aren't worth doing until we run into this situation.

Reply:
Agreed. We can wait until we get there.